src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -466,19 +484,19 @@ class NBDWatchCtx : public librbd::UpdateWatchCtx
   }
 };
 
-static int open_device(const char* path, bool try_load_module = false)
+static int open_device(const char* path, Config *cfg = NULL, bool try_load_module = false)
```

Comment:
what is The difference between nullptr and NULL? I see many NULL are used in this file